rpc/types_broadcast_transaction.go

```diff
@@ -0,0 +1,47 @@
+package rpc
+
+import "github.com/NethermindEth/juno/core/felt"
+
+type BroadcastTxn interface{}
+
+var (
+	_ BroadcastTxn = BroadcastInvokev0Txn{}
+	_ BroadcastTxn = BroadcastInvokev1Txn{}
+	_ BroadcastTxn = BroadcastDeclareV1Txn{}
+	_ BroadcastTxn = BroadcastDeclareV2Txn{}
+	_ BroadcastTxn = BroadcastDeployAccountTxn{}
+)
+
+type BroadcastInvokev0Txn struct {
+	InvokeTxnV0
+}
+
+type BroadcastInvokev1Txn struct {
+	InvokeTxnV1
+}
+
+type BroadcastDeclareV1Txn struct {
+	Type TransactionType `json:"type"`
+	// SenderAddress the address of the account contract sending the declaration transaction
+	SenderAddress           *felt.Felt         `json:"sender_address"`
+	MaxFee                  *felt.Felt         `json:"max_fee"`
+	Version                 TransactionVersion `json:"version"`
```

Comment:
This property should be of type `NumAsHex` instead of `TransactionVersion`, which is a string (https://github.com/starkware-libs/starknet-specs/blob/c0980ed9aa5a8eb2def8d9b145fba27b4e29b7f9/api/starknet_api_openrpc.json#L1892)